includes/alerts.inc.php

```diff
@@ -479,6 +480,7 @@ function DescribeAlert($alert)
     $obj['description']  = $device['purpose'];
     $obj['notes']        = $device['notes'];
     $obj['device_id']    = $alert['device_id'];
+    $obj['alerts_id']    = $alerts_id;
```

Comment:
The correct name would be "alert_id" instead of "alerts_id"

Reply:
No problem changing that object name.  There seems to be this namespace reuse for "alert" where it sometimes gets used for rows in the alerts table and sometimes gets used for rows the alert_log table.  I was using alerts to try to avoid that.  Will change.